packages/perseus/src/types.ts

```diff
@@ -561,7 +570,21 @@ export type WidgetProps<
     // Defines the arguments that can be passed to the `trackInteraction`
     // function from APIOptions for this widget.
     TrackingExtraArgs = Empty,
-> = RenderProps & {
+> = RenderProps & UniversalWidgetProps<Rubric, TrackingExtraArgs>;
+
+/**
+ * The props passed to every widget, regardless of its `type`.
+ */
+export type UniversalWidgetProps<
```

Comment:
Perhaps this should be called something like `WidgetPropsFromRenderer` or `RendererWidgetProps`?